pkg/services/store/object/httpobjectstore/utils.go

```diff
@@ -0,0 +1,117 @@
+package httpobjectstore
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/store/object"
+	"github.com/grafana/grafana/pkg/services/store/router"
+)
+
+const (
+	nameFrameField        = "name"
+	kindFrameField        = "kind"
+	pathPathField         = "path" // scope + uid
+	descriptionFrameField = "description"
+	labelsFrameField      = "labels"
+	fieldsFrameField      = "fields"
+	sizeFrameField        = "size"
+	updatedAtFrameField   = "updatedAt"
+	updatedByFrameField   = "updatedBy"
+)
+
+func resultsToFrame(ctx context.Context, rsp *object.ObjectSearchResponse, router router.ObjectStoreRouter) *data.Frame {
```

Comment:
I am really really doubtful that Dataframe is a good HTTP API for the object store (And search). Would so much prefer a well-typed normal HTTP API.

Reply:
yes -- i see a good use for both.  at this stage, the dataframe version is just because we can show a table with all the properties easily :)